docs/Concepts/Privacy/Multi-Tenancy.md

```diff
@@ -0,0 +1,26 @@
+description: Multi-tenancy
+<!--- END of page meta data -->
+
+# Multi-tenancy
+
+By default, each participant in a privacy network uses its own Besu and Orion
+node.
+
+Multi-tenancy allows multiple participants to use the same Besu and
+Orion node. Each participant is called a Tenant, and the Operator is the
+owner of the Besu and Orion node.
+
+!!! important
+    The Operator is responsible for [configuring multi-tenancy](../../Tutorials/Privacy/Configuring-Multi-Tenancy.md), and has 
+    access to all tenant data.
+    
+![Multi-tenancy](../../images/Multi-tenancy.png)
+
+The Orion client API can only be accessed by the multi-tenant Besu node.
```

Comment:
Sorry wasn't clear before, it can be accessed but we want to ensure they configure Orion so that can only be accessed by the multi-tenancy Besu otherwise your data might not be safe. So in a banner or whatever is done to make this standout as important, something along the lines of "The Orion client API should only be accessed by the multi-tenant Besu node. Otherwise other Orion clients including other Besu nodes might be able to access your data. Access to your data is only secured through use of Besu using multi-tenancy mode."

Reply:
I think this needs to go in an *Important* box -> if you get it wrong, bad things can happen.  @jframe - does something like the following reflect what you're saying: 

_Ensure the multi-tenant Orion node client API is configured to allow access only by the multi-tenant Besu node. Access to your data is secured through Besu using multi-tenancy mode._ 

_If not configured to allow access only by the multi-tenant Besu node, other Orion clients including other Besu nodes may be able to access tenant data._